pkgs/top-level/all-packages.nix

```
@@ -38709,11 +38697,7 @@ with pkgs;
 
   btcdeb = callPackage ../applications/blockchains/btcdeb { };
 
-  jami = callPackages ../applications/networking/instant-messengers/jami {
-    # TODO: remove once `udev` is `systemdMinimal` everywhere.
-    udev = systemdMinimal;
-    jack = libjack2;
-  };
+  jami = callPackages ../applications/networking/instant-messengers/jami { };
```

```suggestion
  jami = callPackages ../applications/networking/instant-messengers/jami {
    jack = libjack2;
  };
```
i assume that was removed by accident.